aws_network_tap/models/spile_tapper.py

```diff
@@ -171,13 +172,18 @@ def discover(self) -> Generator[Spile, None, None]:
                         )
 
     @classmethod
-    def manage(cls, region: str, vpc_ids: List[str], target_id: str, blacklist: List[str] = None) -> None:
+    def manage(cls, region: str, vpc_ids: List[str], config: VPCTagConfig) -> None:
         tapper = SpileTapper(region=region, vpc_ids=vpc_ids)
-        ec2_blacklist = Ec2ApiClient.get_blacklist(region=region)
+        if not config.enabled:
+            return
+        ec2_blacklist = ec2_whitelist = []
```

Comment:
This binds both variables to the same list.
```python
Python 3.7.3 (default, Oct  7 2019, 12:56:13) 
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> b = w = []
>>> b.append(1)
>>> b
[1]
>>> w
[1]
>>> 
```
I suggest initializing these variables on separate lines.